src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -108,6 +108,16 @@ const UsersFragment = ({
     parseToken();
   }, []);
 
+  React.useEffect(() => {
+    if (searchText.length === 0) {
+      setActiveSections([]);
+    } else if (searchText.length >= 0 && followedUsers.length > 0) {
+      setActiveSections([0]);
+    } else {
+      setActiveSections([]);
+    }
+  }, [searchText, followedUsers]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The updated `useEffect` hook now sets `activeSections` based on the length of `searchText` and `followedUsers`. This seems like a logical change that could improve the UX by automatically expanding or collapsing sections based on user input and followed users.
```diff
+   React.useEffect(() => {
+     if (searchText.length === 0) {
+       setActiveSections([]);
+     } else if (searchText.length >= 0 && followedUsers.length > 0) {
+       setActiveSections([0]);
+     } else {
+       setActiveSections([]);
+     }
+   }, [searchText, followedUsers]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->